gio/src/auto/file_info.rs

```diff
@@ -2,6 +2,7 @@
 // from gir-files (https://github.com/gtk-rs/gir-files)
 // DO NOT EDIT
 
+use crate::str;
```

Comment:
This one is also strange. 🤔

Reply:
I think gir automatically tries to "include" everything used for `AsRef/IsA` and would need special casing for `str`

Reply:
https://github.com/gtk-rs/gir/blob/395e9ab1395d369c288167684ea42709a9ec592b/src/analysis/bounds.rs#L280 For the record.